Extensions/Console.json

```diff
@@ -0,0 +1,289 @@
+{
+  "author": "Arthur Pacaud (arthuro555)",
+  "description": "A console behavior that allows parsing simple commands inputed via a Text Input object.",
+  "extensionNamespace": "",
+  "fullName": "Console",
```

Comment:
Maybe something like "Console (commands handler for Text Entry objects)" (a bit long, but more precise)